samples/bluetooth/channel_sounding_ras_initiator/src/distance_estimation.c

```diff
@@ -267,9 +196,17 @@ static void process_tone_info_data(struct processing_context *context,
 			return;
 		}
 
+		int antenna_path = bt_le_cs_get_antenna_path(context->n_ap,
+							     antenna_permutation_index,
+							     i);
+		if (antenna_path == -EINVAL) {
```

Comment:
```suggestion
		if (antenna_path < 0) {
```

would be more future-proof